server/src/users/users.controller.ts

```diff
@@ -17,6 +29,16 @@ export class UsersController {
     return user;
   }
 
+  @Post('reset_password')
+  async resetPassword(@Request() req, @Response({ passthrough: true }) response: ResponseT) {
```

Comment:
oh one more thing here, return typing
i think should be `): Promise<void> {` since we're not returning anything
nbd if we don't have it, just a nice to have

Reply:
great. done.